sdk/go/x/auto/local_workspace.go

```diff
@@ -375,16 +354,6 @@ func (l *LocalWorkspace) RemoveStack(ctx context.Context, fqsn string) error {
 // ListStacks returns all Stacks created under the current Project.
 // This queries underlying backend and may return stacks not present in the Workspace (as Pulumi.<stack>.yaml files).
 func (l *LocalWorkspace) ListStacks(ctx context.Context) ([]StackSummary, error) {
-	user, err := l.WhoAmI(ctx)
```

Comment:
What's the thinking behind removing the whoami and stack list checks?

Reply:
Previously this API made an effort to normalize the returned stack name into either `user/proj/stack` or `org/proj/stack`. We've dropped that logic as a part of this change given that the local backend behavior is problematic and it's safer to just trust what the CLI gives us back.

As a result, the info from whomai/project are no longer needed here.